src/components/ReorderableListCore/useReorderableListCore.ts

```diff
@@ -124,6 +124,8 @@ export const useReorderableListCore = <T>({
   const duration = useSharedValue(animationDuration);
   const scaleDefault = useSharedValue(1);
   const opacityDefault = useSharedValue(1);
+  const dragDirection = useSharedValue(0);
+  const lastDragDirectionPivot = useSharedValue<null | number>(null);
```

Comment:
opted for `null` as the sentinel case because `-1` is a plausible value for a coordinate position.